app/views/reports/regions/show.html.erb

```diff
@@ -300,4 +300,5 @@
 <div id="data-json" style="display: none;">
   <%= raw @data.to_json %>
 </div>
+<script src="//cdnjs.cloudflare.com/ajax/libs/numeral.js/2.0.6/numeral.min.js"></script>
```

Comment:
What happens if this asset fails to load, eg if the CDN is down/slow? Does the page no longer display numbers? Perhaps we could define a placeholder `numeral` that returns the plain number?

```js
var numeral = function(value) {
  return {
    format: function() { return value }
  }
}
```

that can get overridden once the library loads.

Reply:
@harimohanraj89 thanks, that's a great point I didn't think of. Do I just do a `yarn install numeral`?

Reply:
You can run `bin/yarn add numeral` (and also `bin/yarn add chart.js` if you'd like), and then "require" them in `application.js` just like we [require tablesort](https://github.com/simpledotorg/simple-server/blob/master/app/assets/javascripts/application.js#L19).

Reply:
PS hit me up if you wanna pair on this later today.

Reply:
> You can run `bin/yarn add numeral` (and also `bin/yarn add chart.js` if you'd like), and then "require" them in `application.js` just like we [require tablesort](https://github.com/simpledotorg/simple-server/blob/master/app/assets/javascripts/application.js#L19).

@harimohanraj89 I tried requiring the `chart.js` module as well but Rails expects a file named `chart.js` to exist in our js directory when I add `//= require chart.js` (that's the name of the module...). I did a quick search online, but couldn't find a fix. Do you know of an easy fix for this? Or should we punt and change chart.js in the future?

Reply:
I think you can skip adding chart.js to the asset pipeline for now, that can be a separate PR.